markets/bfp-market/contracts/modules/MarketConfigurationModule.sol

```diff
@@ -53,6 +54,10 @@ contract MarketConfigurationModule is IMarketConfigurationModule {
         // Only allow an existing per market to be configurable. Ensure it's first created then configure.
         PerpMarket.exists(marketId);
 
+        if (data.skewScale == 0) {
+            revert ErrorUtil.ZeroAmount();
```

Comment:
I want to add more configuration validation checks here. Can we use a `error InvalidParameter(string parameter, string reason);` here instead?

Reply:
Yep cool I was thinking about that too

Reply:
Done!